docs/common-coreboot-docs/dumping_logs.md

```diff
@@ -7,20 +7,27 @@ a running system.
 
 ## System information
 
-One may use [Dasharo Tools Suite HCL report](../../dasharo-tools-suite/documentation#hcl-report)
+One may use [Dasharo Tools Suite HCL report](../dasharo-tools-suite/documentation#hcl-report)
```

Comment:
@miczyg1 I have a problem with this link. I just built the doc locally and this one redirects me to the empty site

Reply:
Thanks, there were two broken DTS links in this file actually. Both fixed @sulewskiprzemyslaw